esp/templates/survey/review.tex

```diff
@@ -11,25 +11,37 @@
 
 \vspace{0.3in}
 
-{% if not surveys|length_is:0 %}
-Students from {{ prog.niceName }} filled out an online survey asking some questions about the program as a whole.  Their responses are summarized below.
-{% endif %}
-
 {% for s in surveys %}
 \vspace{0.25in}
 {% if not forloop.first %}\clearpage{% endif %}
     \begin{center}
-    \begin{longtable}{|l|l|} \hline
-    \multicolumn{2}{|c|}{\cellcolor{esphead} ~ } \\
-    \multicolumn{2}{|c|}{\cellcolor{esphead} \Large {Responses for {{ s.name }}} } \\ 
-    \multicolumn{2}{|c|}{\cellcolor{esphead} ~ } \\\hline
+    \begin{longtable}{|p{0.5\linewidth}|p{0.5\linewidth}|} \hline
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} \Large {Responses for {{ s.name }}} } \\ 
+    \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\\hline
     {% for q in s.display_data.questions %}
         {% with s.num_participants as num_participants %}
             {% include "survey/answers.tex" %}
         {% endwith %}
     {% endfor %}
     \end{longtable}
     \end{center}
+    {% for c in s.perclass_data %}
+        \clearpage
+        \begin{center}
+        \begin{longtable}{|p{0.5\linewidth}|p{0.5\linewidth}|} \hline
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} \Large {Responses for {{ c.class.emailcode }}: {{ c.class.title|texescape }}} } \\ 
+        \multicolumn{2}{|C{\linewidth}|}{\cellcolor{esphead} ~ } \\\hline
+        \multicolumn{2}{|C{\linewidth}|}{Teachers: {{ c.class.parent_class.getTeacherNames|join:", "|texescape }} } \\ \hline
+        {% for q in c.questions %}
```

Comment:
Can you add a little padding above the question name? and above the teacher names while you're at it too... just to be nicer on the eyes. 
![image](https://user-images.githubusercontent.com/25392075/79692990-2fcc8980-8236-11ea-95dd-036f5cf2adfd.png)   ![image](https://user-images.githubusercontent.com/25392075/79692997-36f39780-8236-11ea-9d64-4fc153fe58c0.png)

Reply:
I added some padding-- lmk if it looks good on your end. Note that the histogram is still not well padded but when i tried to add padding it threw off the other column and such so I just left it for now. The other stuff should look much better though.